circle.yml

```diff
@@ -13,7 +13,14 @@ deployment:
   staging:
     branch: staging
     commands:
+<<<<<<< HEAD
```

Comment:
What is this? It should not be here, looks like you have rebased or had some git conflicts. You should fix this.